terragrunt/aws/ecr/iam_oidc.tf

```
@@ -98,4 +98,25 @@ data "aws_iam_policy_document" "ecr_image_manage" {
     ]
     resources = ["*"]
   }
+}
+
+data "aws_iam_policy_document" "pr_review_env_policy_document" {
```

Can you create a new policy that uses this policy document and attaches it to the role like so:
```suggestion
resource "aws_iam_role_policy_attachment" "pr-review-env-lambda" {
  role       = aws_iam_role.pr-review-env-lambda.name
  policy_arn = aws_iam_policy.pr_review_env.arn
}
resource "aws_iam_policy" "pr-review-env-lambda" {
  name   = "pr-review-env-lambda"
  path   = "/"
  policy = data.aws_iam_policy_document.pr_review_env_policy_document.json
}

data "aws_iam_policy_document" "pr_review_env_policy_document" {
```